config/concourse/_ytt_lib/concourse/rendered.yml

```diff
@@ -43,6 +43,9 @@ spec:
   - name: atc
     port: 80
     targetPort: atc
+  - name: atc-tls
```

Comment:
@yatzek Is this the correct file? Shouldn't it be added in an overlay file?

Reply:
This should be fine, the actual change is in the "build/concourse/values.yml" file and the build step produces "config/concourse/_ytt_lib/concourse/rendered.yml" file, there is no need to do any ytt mods here.